app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) == 3:
+        start_command, origin_file, new_file = command
+        if start_command == "mv" and "/" not in new_file:
```

Comment:
it is bad practice to check for "/" to determine whether there is directory in path or not
Use os.path.split